fhd_core/setup_metadata/fhd_struct_update_obs.pro

```diff
@@ -50,8 +50,8 @@ struct.dimension=Float(dimension)
 struct.elements=Float(elements)
 struct.kpix=Float(kbinsize)
 struct.degpix=Float(degpix)
-struct.max_baseline=Float(max_baseline)
-struct.min_baseline=Float(min_baseline)
+struct.max_baseline=max_baseline
+struct.min_baseline=min_baseline
```

Comment:
As above, if these are allowed to sometimes be doubles, we have to force them to always be doubles.